server/test/unit/paTest/getGooglePayloadTest.js

```diff
@@ -0,0 +1,100 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const { helpers, googleHelpers } = require('brave-alert-lib')
+const { mockResponse } = require('../../../testingHelpers')
+const pa = require('../../../pa')
+
+use(sinonChai)
+
+const sandbox = sinon.createSandbox()
+
+const fakePaPayload = {
+  aud: 'fakeclientid.apps.googleusercontent.com',
+  iss: 'https://accounts.google.com',
+  exp: Date.now(),
+  hd: 'brave.coop',
+  email: 'brave@brave.coop',
+  name: 'Brave User',
+}
+
+describe('pa.js unit tests: getGooglePayload', () => {
+  beforeEach(() => {
+    sandbox.spy(helpers, 'log')
+    sandbox.spy(pa, 'getGooglePayload')
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('for a valid ID token', () => {
+    beforeEach(async () => {
+      // have googleHelpers.paGetPayload return successfully
+      sandbox.stub(googleHelpers, 'paGetPayload').returns(fakePaPayload)
+
+      this.res = mockResponse(sandbox)
+      await pa.getGooglePayload({ body: { idToken: 'id-token' } }, this.res)
+    })
+
+    it('should respond with status 200 (OK)', () => {
+      expect(this.res.status).to.be.calledWith(200)
+    })
+
+    it('should respond with JSON data aud, iss, exp, hd, email, and name, as returned by paGetPayload', () => {
+      expect(this.res.json).to.be.calledWith(fakePaPayload)
+    })
+
+    it('should not log any information', () => {
+      expect(helpers.log).to.not.be.called
+    })
+  })
```

Comment:
Great job! Seems like all the possibilities are covered and the tests were super clear and easy to follow.